### ISSUE TITLE
Import Remote Model to Workspace gives error git-upload-pack not permitted on https://....

### ISSUE THREAD

**Issue by svendebie** [archimatetool/archi-modelrepository-plugin#117]
### Error log
- The error log is empty.

### Version of Archi, Operating System
Version 4.6.0 on Windows 7 64-bits.

### Expected Behaviour
My Archimate model is connected to my private GIT repository on Bitbucket and I can collaborate with everyone who has access to that repository and push/fetch changes to that repo.

### Actual Behaviour
I get this error:
![image](https://user-images.githubusercontent.com/27744969/72724367-6a654e00-3b83-11ea-9439-358d49b448e2.png)

### Steps to Reproduce the Behaviour
1. Go to menu > Help > Reset Workbench to Defaults...
2. Go to menu > Collaboration > Import remote model to workspace
3. And fill in my private repository on bitbucket
![image](https://user-images.githubusercontent.com/27744969/72724581-042cfb00-3b84-11ea-9e5e-42eb53c991b4.png)
4. After pressing OK I get the error message as shown with Actual behaviour

**Comment by Phillipus** [comment]
The error message is a generic one reported by JGit and it doesn't help, unfortunately. It could be an authentication error due to your particular setup, credentials and/or certification. Might be worth searching the issue tracker here.

**Comment by jbsarrodie** [comment]
My 2cts: either you have to setup a proxy in Archi and you didn't, or you define one but you shouldn't.

**Comment by JereMist** [comment]
Hello, I encountered the same error. I checked proxy and it is not the problem. Did you fixed it ?

**Comment by Phillipus** [comment]
@JereMist The error can occur depending on different factors. Please provide more details about your setup, versions, how you are connecting, proxy, etc...

**Comment by jbsarrodie** [comment]
> Did you fixed it ?

In fact (unfortunately?) there is nothing to fix in the plugin: this is linked to a configuration misalignment between the collaboration plugin and your git server (or something in between).

Some topics to check:
- Proxy configuration (if in use)
- Git server using a selfsigned certificate or one from a custom PKI (in such case certificate must be manually added to the JVM's truststore)
- Access right in Git platform
- Git platform access management based on user/password on the web frontend but requiring a token for repositories
- Git platform access management based on SSO on the web frontend but requiring a user/password or a token for repositories
- In some occasion, JGit (the java based implementation of git) can try to use your local and native git client, leading to some issues.
- I've seens some issues related to MTU when working through a VPN

If you google _git-upload-pack not permitted_, you'll see many cases where this error message appears (but unfortunately few simple answers).

**Comment by JereMist** [comment]
With your answers I checked on my configuration, and finally I fixed my problem : I did not enable SSO on my token (with my company account).

Like jbsarrodie said, it was a problem on my configuration. Many thanks !

**Comment by jbsarrodie** [comment]
> Many thanks !

You're welcome.

I guess the real fix here is to create a wiki page for this kind of issues

**Comment by Phillipus** [comment]
> I guess the real fix here is to create a wiki page for this kind of issues

I'm already on it...

**Comment by Phillipus** [comment]
> 
> 
> > I guess the real fix here is to create a wiki page for this kind of issues
> 
> I'm already on it...

I created this wiki page - https://github.com/archimatetool/archi-modelrepository-plugin/wiki/Troubleshooting-Connection-Issues

I've added the reply from @jbsarrodie and some intro text. It's a start, so we can add to it and expand on it as issues are solved. If users solve their connection issues (like this one) we can add details to the wiki page and reference the issue.
